client/src/components/Workflow/Editor/Lint.vue

```diff
@@ -1,470 +1,237 @@
-                                        />
-                                    </span>
-                                </li>
-                            </ul>
-                            <em class="small">
-                                <p>
-                                    Inputs to tool and subworkflow steps should be connected to formal workflow inputs.
-                                    {{ whyInputs }}
-                                </p>
-                                <p>
-                                    Most such step inputs can be automatically connected via Galaxy, click the
-                                    <font-awesome-icon icon="cog" /> for inputs to have Galaxy attempt this.
-                                </p>
-                            </em>
-                        </div>
-                    </span>
-                </LintSection>
-                <LintSection :okay="inputsMetadataOkay" title="Input Metadata" collapse-id="c-input-metadata">
-                    <span>
-                        <div v-if="inputsMetadataOkay">
-                            All workflow inputs have labels and annotations.
-                        </div>
-                        <div v-else>
-                            <p>
-                                Some workflow inputs are missing labels and/or annotations.
-                            </p>
-                            <ul>
-                                <li
-                                    v-for="(input, idx) in inputsMissingMetadata"
-                                    :key="idx"
-                                    @mouseover="highlight(input.stepId)"
-                                    @mouseleave="unhighlight(input.stepId)"
-                                >
-                                    <b @click="scrollTo(input.stepId)" class="scrolls"
-                                        ><i :class="input.stepIconClass" />{{ input.stepLabel }}</b
-                                    >
-                                    <em class="small">
-                                        <span class="input-missing-metadata-summary">
-                                            This input is {{ missingSummary(input) }}.
-                                        </span>
-                                    </em>
-                                </li>
-                            </ul>
-                        </div>
-                    </span>
-                </LintSection>
-                <LintSection :okay="outputsOkay" title="Outputs" collapse-id="c-outputs">
-                    <span>
-                        <div v-if="outputsOkay">
-                            This workflow has outputs and they all have valid labels.
-                        </div>
-                        <div v-else-if="outputs.length == 0">
-                            This workflow has no labeled outputs, please select and label at least one output.
```

Comment:
We're dropping this warning right? Can this be restored?

Reply:
Add the additional warning.